syncd/Syncd.cpp

```diff
@@ -898,6 +959,122 @@ sai_status_t Syncd::processEntry(
     }
 }
 
+sai_status_t Syncd::processBulkOidCreate(_In_ sai_object_type_t objectType,
+            _In_ sai_bulk_op_error_mode_t mode,
+            _In_ const std::vector<std::string>& objectIds,
+            _In_ const std::vector<std::shared_ptr<saimeta::SaiAttributeList>>& attributes,
+            _Out_ std::vector<sai_status_t>& statuses)
+{
+    SWSS_LOG_ENTER();
+
+    /*
+     * Switch, Port objects doesn't have bulk API`s, so we no need additional
```

Comment:
a lot of other objects also don't have bulk API, this check is not needed, please remove, should be done in meta/Meta.cpp anyway